crates/sui-types/src/messages_consensus.rs

```diff
@@ -24,16 +22,29 @@ use std::hash::{Hash, Hasher};
 use std::sync::Arc;
 use std::time::{SystemTime, UNIX_EPOCH};
 
+/// The index of an authority in the consensus committee.
+/// The value should be the same in Sui committee.
+pub type AuthorityIndex = u32;
+
+/// Consensus round number.
+pub type Round = u32;
+
+/// The index of a transaction in a consensus block.
+pub type TransactionIndex = u16;
+
+/// Consensus timestamp in ms.
+pub type ConsensusTimestampMs = u64;
```

Comment:
can we be consistent about whether or not we are prefixing consensus-related type aliases with `Consensus`?

that is to say, should it be removed here or added on the types above?

Reply:
I was trying to avoid confusion with `CheckpointTimestamp` or other timestamp types. But looking at how the Narwhal imported `TimestampMs` type is used without issue, I will leave this as `TimestampMs` for now.